docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -83,59 +83,40 @@ WARNING: This setting potentially allows a link:https://en.wikipedia.org/wiki/Cr
 
 When starting a new server within a component, {prod-short} autodetects this, and the UI offers to automatically expose this port as a `public` port. This is useful for debugging a web application, for example. It is impossible to do this for servers that autostart with the container (for example, a database server). For such components, specify the endpoints explicitly.
 
-Example specifying endpoints for `kubernetes`/`openshift` and `plugin` types:
+Example specifying endpoints for `kubernetes`/`openshift` types:
 
 [source,yaml]
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
 components:
   - name: theia-editor
-    plugin:
-      id: eclipse/che-theia/next
       endpoints:
       - name: 'theia-extra-endpoint'
         targetPort: 8880
-        configuration:
-          discoverable: true
-          public: true
-
-  - plugin:
-      id: redhat/php/latest
-      memoryLimit: 1Gi
-      endpoints:
-      - name: 'php-endpoint'
-        targetPort: 7777
-
-  - plugin:
-      name: theia-editor
-      id: eclipse/che-theia/next
-      endpoints:
-      - name: 'theia-extra-endpoint'
-        targetPort: 8880
-        configuration:
+        exposure:
```

Comment:
this is not the correct format, pls check https://docs.devfile.io/devfile/2.0.0/user-guide/api-reference.html

exposure is a property which has a value. And discoverable, public, scheme are not valid properties